lesson 3/index.html

```diff
@@ -0,0 +1,151 @@
+<head>
+    <meta charset="UTF-8">
+    <title>Lesson 3</title>
+</head>
+<body>
+<img src="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcQQzzQZBaltEAjk04xq6JAc_uacME4oZpEajg&usqp=CAU" alt="html picture">
+
+<ol>
+    <li> Знакомство с Web
+        <ul>
+            <li>Краткая история Web и что такое интернет.</li>
+            <li>Как устроен браузер и как он получает страницы по сети.</li>
+            <li>Что такое Git?</li>
+            <li>Создание первого репозитория.</li>
+            <li>О фацле .gitignore.</li>
+        </ul>
+    </li>
+    <li>HTML-верстка - первые шаги
+        <ul>
+            <li>Настройка рабочей среды.</li>
+            <li>Краткая история HTML.</li>
+            <li>Какие бывают теги и что такое атрибут.</li>
+            <li>Теги для работы с текстом.</li>
+            <li>Создание собственной страницы.</li>
+        </ul>
+    </li>
+    <li>Погружение в HTML-разметку
+        <ul>
+            <li>Мета-теги.</li>
+            <li>Мнемоники.</li>
+            <li>Тег для работы с изображениями.</li>
+            <li>Теги для работы со списками.</li>
+            <li>Теги для работы с таблицами.</li>
+            <li>Теги для работы с формами.</li>
+            <li>Div-блоки.</li>
+        </ul>
+    </li>
+</ol>
+<hr>
+
+<table border="1" width="60%">
+    <thead>
+    <tr>
+        <th>Tag</th>
+        <th>Description</th>
+    </tr>
+    </thead>
+    <tbody>
+    <tr>
+        <td>&lttable&gt</td>
+        <td>Defines a table</td>
+    </tr>
+    <tr>
+        <td>&lttr&gt</td>
+        <td>Defines a row in a table</td>
+    </tr>
+    <tr>
+        <td>&ltth&gt</td>
+        <td rowspan="2">Defines a "header cell"/"cell" in a table</td>
+    </tr>
+    <tr>
+        <td>&lttd&gt</td>
+
+    </tr>
+    <tr>
+        <td colspan="2">Rarely used</td>
+
+    </tr>
+    <tr>
+        <td>&ltcaption&gt</td>
+        <td>Defines a table caption</td>
+    </tr>
+    <tr>
+        <td>&ltthead&gt</td>
+        <td>Groups the header content in a table</td>
+    </tr>
+    <tr>
+        <td>&lttbody&gt</td>
+        <td>Groups the body content in a table</td>
+    </tr>
+    <tr>
+        <td>&lttfoot&gt</td>
+        <td>Groups the footer content in a table</td>
+    </tr>
+    </tbody>
+
+</table>
+<hr>
+
+<form>
+    <p>
+        <label for="name">Name</label><br>
+        <input id="name" name = "name" type="text" placeholder="Name"/><br>
+    </p>
+    <p>
+        <label for="e-mail">E-mail</label><br>
+        <input id="e-mail" name = "e-mail" type="email"/><br>
+    </p>
+    <p>
+        <label for="Password">Password</label><br>
+        <input id="Password" name = "Password"/>
+    </p>
+
+    <p>
+        <label for="programming">Your programming level</label><br>
+        <input id="programming" name="level" type="range" min="0" max="10" value="5">
+    </p>
+
+    <p>
+        What does HTML stand for?<br>
+        <input id="I" name="HTML abbreviation" type="checkbox" value="1">
```

Comment:
Should you find a better name for id? The best case will be, if you can undestand what is this from the name 
Try to escape space inside name, for example html_abbreviation